DEVELOPMENT.md

```diff
@@ -29,86 +42,87 @@ flowchart LR
 
-In addition to the usual requirements, the integration tests make use of a GraphQL-enabled Buildkite API token in order to create pipelines, start builds, read logs, and clean up afterwards.
+### Setup
 
-## Setup
+Any Buildkite user who has an access to a Kubernetes cluster should be able to run our integration test.
 
-Any member of the public should be able to run our integration as long as you are an user of Buildkite, and you have
-access to a Kubernetes cluster.
+To get the integration test running locally, you will need:
 
-Concretely, to get the integration test running locally, you will need:
-1. A valid Buildkite API token (presuming you are a customer of Buildkite).
-2. A valid Buildkite Agent Token in your target Buildkite Cluster. The agent token needs to be installed in your K8s
-   cluster.
-3. Your organization name in Buildkite and your target Buildkite Cluster UUID.
-4. Depending on test cases, you may also need a SSH keys, please read below.
-5. Your shell environment will need CLI write access to a k8s cluster such as the one provided by https://orbstack.dev/.
+1. A valid Buildkite API token with GraphQL enabled.
+2. A valid Buildkite Agent Token in your target Buildkite Cluster.
+3. The name of your Buildkite organization (slug) and your target Buildkite Cluster UUID.
+4. Depending on test cases, you may also need SSH keys, please keep reading.
+5. Your shell environment will need CLI write access to a Kubernetes cluster such as the one provided by https://orbstack.dev/.
 
-### Use environment variables
+#### Using environment variables
 
-We found it's convenient to supply API token, organization name, and cluster UUID as environment variables. This can be done using an `.envrc` file loaded by using [direnv](https://direnv.net/).
+It's generally convenient to supply the API token, your Buildkite organization name, and cluster UUID as environment variables. This can be done using an `.envrc` file loaded by using [direnv](https://direnv.net/).
 
 ```bash
 export BUILDKITE_TOKEN="bkua_**************"
 export ORG="your-cool-org-slug"
 export CLUSTER_UUID="UUID-UUID-UUID-UUID"
 ```
 
-# Running Locally
+## Running locally
 
-To run the controller locally, with the environment variables, note in this example I am overriding the queue to ensure jobs from the default queue which is "", are picked up by this agent.
+To run the controller locally, with the environment variables, run the following example. Note that in this example the queue is overridden to ensure jobs from the default queue, which is "", are picked up by the Buildkite agent.
 
-```
+```bash
 just run --org $ORG --buildkite-token $BUILDKITE_TOKEN --debug --tags 'queue=,os=linux'
 ```
 
-# Testing Locally
+## Testing locally
 
-Before you start, check which kubernetes cluster configuration your using by default.
+Before you start, check which Kubernetes cluster configuration you are using by default:
 
-```
+```bash
```

Comment:
There were a series of these code blocks missing the `bash` syntax styler. I hope you don't mind me adding these to the code blocks that were missing them as my IDE's linter tool kept pointing them out to me.

Anyway, I guess this makes them more consistent with the other code blocks on this page.